build_tools/update_environments_and_lock_files.py

```diff
@@ -211,6 +211,69 @@ def remove_from(alist, to_remove):
         + ["wheel", "pip"],
         "package_constraints": {"python": "3.8", "blas": "[build=mkl]"},
     },
+    {
+        "build_name": "doc_min_dependencies",
+        "folder": "build_tools/circle",
+        "platform": "linux-64",
+        "channel": "conda-forge",
+        "conda_dependencies": remove_from(
+            common_dependencies_without_coverage, ["matplotlib"]
+        )
+        + [
+            "memory_profiler",
+            "compilers",
+            "sphinx",
+            "sphinx-gallery",
+            "numpydoc",
+            "sphinx-prompt",
+        ],
+        # scikit-image min version (0.14.3) not available through conda-forge
```

Comment:
Can we bump scikit-image to 0.16.2, which has a version on conda-forge for py38?

From [scikit-image's pypi](https://pypi.org/project/scikit-image/#history) release history, 0.14.3 was released on 6/11/19, and 0.16.2 was on 10/22/19. Our min supported numpy 1.17.3 was released on [10/17/19](https://pypi.org/project/numpy/1.17.3/).